index.html

```diff
@@ -330,33 +330,37 @@ <h2>Definitions</h2>
 
   <section id="data-model">
       <h2>DAPT Data Model and corresponding TTML syntax</h2>
-      <p>This section specifies the data model for DAPT and its corresponding TTML syntax.</p>
+      <p>This section specifies the data model for DAPT and its corresponding TTML syntax. In the model, there are objects which can have properties and be associated with other objects. In the TTML syntax, these objects and properties are expressed as elements and attributes.</p>
       <p class="issue" data-number="12"></p>
       <section>
         <h3>DAPT Script</h3>
-          <p>A <dfn>DAPT Script</dfn> corresponds to a document processed within an authoring workflow. It has  properties defined in the following sections: <a>Script Type</a>, <a>Primary Language</a>, <a>Events</a> and, for Dubbing Scripts, <a>Characters</a>. It is represented as a TTML document with the structure and constraints also defined in the following sections.</p>
+          <p>A <dfn>DAPT Script</dfn> corresponds to a document processed within an authoring workflow or processed by a client. It has properties defined in the following sections: <a>Script Type</a>, <a>Primary Language</a>, <a>Script Events</a> and, for Dubbing Scripts, <a>Characters</a>.</p>
+          <p>A <a>DAPT Script</a> is represented as a TTML document with the structure and constraints also defined in the following sections.</p>
```

Comment:
Suggest defining "TTML document" as a TTML2 Document Instance, in the definitions section.

Reply:
https://github.com/w3c/dapt/issues/57